packages/sources/tradermade-test/src/config/index.ts

```diff
@@ -0,0 +1,27 @@
+export const customSettings = {
+  API_ENDPOINT: {
+    description: 'API endpoint for tradermade',
+    default: 'https://marketdata.tradermade.com/api/v1/live',
+    type: 'string',
+  },
+  API_KEY: {
+    description:
+      'An API key that can be obtained from [here](https://marketdata.tradermade.com/docs/restful-api)',
+    type: 'string',
+  },
```

Comment:
Based on v2, I think this should be marked as `required: true`. We should also mark it as sensitive.